traitcuration/traits/datasources/ols.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+BASE_URL = "https://www.ebi.ac.uk/ols/api/"
+
+
+def make_ols_query(term_uri, ontology):
+    """
+    Takes in a term URI (or IRI as referenced in OLS documentation) and the ontology id to search against. Returns a
+    dictionary for that term with the fields 'label', 'obo_id' which is the CURIE and 'status'.
+    """
+    results = requests.get(
+        f"{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}").json()
+    print(f"{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}")
+    if 'error' in results:
+        return None
+    term_info = results["_embedded"]["terms"][0]
+    term_curie = term_info["obo_id"]
+    term_label = term_info["label"]
```

Comment:
When parsing a response, it's useful to provide example values in comments to make it more readable. E.g.:

```python
term_curie = term_info["obo_id"]  # E.g. 'MONDO:123456'
term_label = term_info["label"]  # E.g. 'Some disease name'
```